libraries/stdlib/common/src/kotlin/TextH.kt

```diff
@@ -270,3 +270,13 @@ expect fun Long.toString(radix: Int): String
 internal expect fun checkRadix(radix: Int): Int
 
 internal expect fun digitOf(char: Char, radix: Int): Int
+
+/**
+ * Converts the characters in the specified array to a string.
+ */
+expect inline fun String(chars: CharArray): String
+
+/**
+ * Converts the characters from a portion of the specified array to a string.
+ */
+expect inline fun String(chars: CharArray, offset: Int, length: Int): String
```

Comment:
`inline` is an implementation detail of these functions, which may be or may not be suitable on a particular platform.
There's no reason to require it to be inline in the expectation.